lib/IRGen/GenConstant.cpp

```diff
@@ -0,0 +1,131 @@
+//===--- GenStruct.cpp - Swift IR Generation For 'struct' Types -----------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file implements IR generation for constant values.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/IntrinsicInst.h"
+#include "llvm/IR/InlineAsm.h"
+#include "llvm/IR/Intrinsics.h"
+#include "llvm/ADT/MapVector.h"
+#include "llvm/ADT/SmallBitVector.h"
+#include "llvm/ADT/TinyPtrVector.h"
+
+#include "llvm/IR/Constant.h"
+#include "IRGenModule.h"
+#include "GenStruct.h"
+#include "TypeInfo.h"
+
+using namespace swift;
+using namespace irgen;
+
+/// Construct a ConstantInt from an IntegerLiteralInst.
+llvm::Constant *IRGenModule::emitConstantInt(swift::IntegerLiteralInst *ILI) {
```

Comment:
These three functions were basically just moved from `IRGenSIL.cpp`, and still get called from there (`IRGenSILFunction::visit{Integer,Float,String}LiteralInst` respectively).

(The `swift` qualifier just came from the original code, but obviously I can still fix it.)